.pipelines/OneBranch.Official.yml

```diff
@@ -83,9 +83,16 @@ extends:
             set PYTHONPATH=
             set PYTHONHOME=
             python -m pip install --upgrade pip
-            python -m pip install cibuildwheel
+            python -m pip install cibuildwheel numpy
             python -m cibuildwheel --platform windows --output-dir $(REPOROOT)\out
           displayName: Build wheels
+        - task: SDLNativeRules@3
```

Comment:
Do not let this task override the files in `$(REPOROOT)\out`. Please consider to use a different folder. Or you may upload the built python packages first.

Reply:
This PR doesn't modify files in ```$(REPOROOT)\out```. The output for static analysis is saved in ```windows_out``` folder.